src/types/channelsChartData.ts

```diff
@@ -0,0 +1,5 @@
+export interface ChannelsChartData {
```

Comment:
It does not make sense separate channelsChartData.ts and channelsData.ts. It should be on file called channels.ts